internal/controller/frrconfiguration_controller.go

```diff
@@ -156,15 +156,15 @@ func (r *FRRConfigurationReconciler) Reconcile(ctx context.Context, req ctrl.Req
 	return ctrl.Result{}, nil
 }
 
-func (r *FRRConfigurationReconciler) applyEmptyConfig(req ctrl.Request) error {
+func (r *FRRConfigurationReconciler) ApplyEmptyConfig() error {
```

Comment:
Let's add the req to the external error if it fails

Reply:
done